src/pages/Sketch/components/Canvas.js

```diff
@@ -1,7 +1,7 @@
 import React, {useState, useEffect, useRef, useCallback} from 'react';
 import styles from './Canvas.module.css';
 import Toolbox from './Toolbox/Toolbox';
-import {FaPencilAlt, FaRegSquare, FaDownload, FaRegCircle, FaSlash, FaRegMoon, FaSun, FaFont} from 'react-icons/fa';
+import {FaPencilAlt, FaRegSquare, FaDownload, FaRegCircle, FaSlash, /*FaRegMoon, FaSun,*/ FaFont} from 'react-icons/fa';
```

Comment:
ok I will remove these two icons. Bcz adding these again (when restoring darktheme) is not such a big problem